components/usage/pkg/apiv1/usage_test.go

```diff
@@ -297,8 +299,132 @@ func TestReportGenerator_GenerateUsageReport(t *testing.T) {
 
 	require.Equal(t, nowFunc(), report.GenerationTime)
 	require.Equal(t, startOfMay, report.From)
-	require.Equal(t, startOfJune, report.To)
+	// require.Equal(t, startOfJune, report.To) TODO(gpl) This is not true anymore - does it really make sense to test for it?
```

Comment:
Hmm, you're right. If we're looking at the usage from May to June, and `now` is some time in May, then `to` should be equal to `now` (and not `startOfJune`, which would only be the case if `now` is some time in June or later).

Reply:
There are other potential solutions, but this was the least invasive - and seems to be in line with what we expect at other places. Happy to turn this one around when Milan/Andrew come back (and do better filtering on the instances we retrieve, for example) - as long as the test passes! :wink: